packages/transit-vehicle-overlay/src/styled.ts

```diff
@@ -0,0 +1,154 @@
+import memoize from "lodash.memoize";
+import coreUtils from "@opentripplanner/core-utils";
+import { FC } from "react";
+import styled, { css } from "styled-components";
+
+import { VehicleComponentProps } from "./types";
+
+interface RouteColorBackgroundOptions {
+  /**
+   * The alpha component of a color in hexadecimal.
+   */
+  alphaHex?: string;
```

Comment:
Could we possibly use alphaHex in a story?

Reply:
It is used in the route color on-hover background. Should there be a separate story?

Reply:
Oh I missed that sorry! Yes then we're good to go.